apps/android/package.json

```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@fluentui-react-native/tester": "^0.13.7",
     "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-test-app": "^0.2.1"
+    "react-native": "^0.62.2",
```

Comment:
Probably want to be explicit with which version we pull in? Reproducing and debugging dependency issues is way easier when we have the exact version specified